cmd/internal/config.go

```diff
@@ -17,20 +17,15 @@ limitations under the License.
 package internal
 
 import (
-	"fmt"
 	"log"
 	"os"
 
 	"sigs.k8s.io/kubebuilder/internal/config"
 )
 
-const (
-	noticeColor = "\033[1;36m%s\033[0m"
-)
-
-// ConfiguredAndV1 returns true if the project is already configured and it is v1
-func ConfiguredAndV1() bool {
-	projectConfig, err := config.Read()
+// IsConfigured returns true if the project is already configured
+func IsConfigured() bool {
```

Comment:
I guess it's easier to follow if using `ProjectFileExists`.

Reply:
Renamed to `HasProjectFile` since return a bool.